src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpIntelliSense.cs

```diff
@@ -168,6 +168,7 @@ public void CtrlAltSpaceOption(bool showCompletionInArgumentLists)
             VisualStudio.Editor.Verify.CurrentLineText("namespace Goo$$", assertCaretPosition: true);
 
             ClearEditor();
+            Assert.False(VisualStudio.Editor.IsCompletionActive());
```

Comment:
📝 @JoeRobich I added this assertion before making any of the other changes. When it started failing, I knew the test was reaching an unexpected state, at which point fixing it was easy.

Reply:
ah, interesting!